sql/status_effects.sql

```diff
@@ -374,10 +374,10 @@ INSERT INTO `status_effects` VALUES (336,'misers_roll',131368,0,0,0,0,0,0,0);
 INSERT INTO `status_effects` VALUES (337,'companions_roll',131368,0,0,0,0,0,0,0);
 INSERT INTO `status_effects` VALUES (338,'avengers_roll',131368,0,0,0,0,0,0,0);
 INSERT INTO `status_effects` VALUES (339,'naturalists_roll',131368,0,0,0,0,0,0,0);
-INSERT INTO `status_effects` VALUES (340,'warriors_charge',37,0,0,0,0,0,0,0);
+INSERT INTO `status_effects` VALUES (340,'warriors_charge',36,0,0,0,0,0,0,0);
 INSERT INTO `status_effects` VALUES (341,'formless_strikes',32,0,0,0,0,0,0,0);
-INSERT INTO `status_effects` VALUES (342,'assassins_charge',37,0,0,0,0,0,0,0);
-INSERT INTO `status_effects` VALUES (343,'feint',35,0,0,0,0,0,0,0);
+INSERT INTO `status_effects` VALUES (342,'assassins_charge',36,0,0,0,0,0,0,0);
+INSERT INTO `status_effects` VALUES (343,'feint',36,0,0,0,0,0,0,0);
```

Comment:
Feint should wear on the next _successful_ attack. The behavior was implemented in https://github.com/DarkstarProject/darkstar/pull/5677, but the submitter must have assumed that the lose-on-attack effect flag was 1 instead of 4.

So Feint's effect flag should just be 32 (since I'm not finding evidence it can be dispelled)